scripts/release.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+VERSION=$1
+
+sed -i.bak "s/## \[Unreleased\]/## \[Unreleased\]\n\n## \[${VERSION}\] - $(TZ=Europe/Krakow date '+%Y-%m-%d')/" CHANGELOG.md
+rm CHANGELOG.md.bak 2> /dev/null
```

Comment:
it seems that we are not using .bak for anything? so maybe we can just not create it, and change in place?

Reply:
MacOS's version of sed doesn't play nicely with just `sed -i '....'` and I've found this to be a solution: https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux